### PR TITLE
NPM release docs

### PR DESCRIPTION
**Pull request by hx-markterry**
#### What does this PR do? (please provide any background)

We tried to use a scoped package name, but some of our systems use npm 1 and the name of this project is too general for a package name on NPM by itself.

To support a NPM release and older browsers, just setting the package scope manually before an NPM release seems the best approach for now.

#### What tests does this PR have?

N/A

#### How can this be tested?

N/A

#### What gif best describes how you feel about this work?

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by hx-markterry** [comment]
I need to add some intro text for this project in the readme (@michaelcarter)

**Comment by notABluesSinger** [comment]
Taken Slot 1️⃣

**Comment by hx-markterry** [comment]
any news on this @notABluesSinger ?

**Comment by notABluesSinger** [comment]
Sorry @hx-markterry, what are you waiting for from me?

**Comment by hxtomprice** [comment]
Taking the other slot

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+
 ## Node Apps
 
 ### `createPrivateRelease.sh`
 
 This will check if the `package.json` version has been updated for a project and then create a new release on github after running `npm build` and commiting any changed release assets in the `dist` directory.
+
+## Releasing to NPM
+
+We currently use a combination of npm version 1 and above, because of this we can not scope the package name in the `package.json`, when a new NPM release is required please add the scope to the package name (`@holidayextras/deployment-helpers`) manually before running `npm publish`.
```
Comment (hxtomprice) [inline README.md]: Is it worth mentioning here that this is automatically handled when using `nodeApps/createPrivateRelease.sh`?

**Comment by hxtomprice** [comment]
:+1:

**Pull request merged**